extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -559,23 +562,27 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 
 	public async migrateLogins(): Promise<Boolean> {
 		try {
+			this._loginMigrationModel.AddNewLogins(this._loginsForMigration.map(row => row.loginName));
+
 			const sourceConnectionString = await this.getSourceConnectionString();
 			const targetConnectionString = await this.getTargetConnectionString();
 			console.log('Starting Login Migration at: ', new Date());
 
-			console.time("migrateLogins")
+			console.time("migrateLogins");
```

Comment:
is it possible to remove the console logging? It's good to have for testing but I don't think we should have it in the prod release, as it can clutter up the console (and make it harder to find real errors, etc)